tensorflow/python/ops/image_ops_impl.py

```diff
@@ -2937,7 +2954,8 @@ def reducer(x):
 
 
 @tf_export('image.ssim')
-def ssim(img1, img2, max_val):
+def ssim(img1, img2, max_val, filter_size = _SSIM_FILTER_SIZE,
+         filter_sigma = _SSIM_SIGMA, k1 = _SSIM_K1, k2 = _SSIM_K2):
```

Comment:
We'd prefer if these underscored constants would be numbers

The space around the equal sign will trip our linter

The docstring says a value for k1 and k2 when it should probably say nothing